asserts/database.go

```diff
@@ -53,6 +57,12 @@ type DatabaseConfig struct {
 	TrustedKeys map[string][]PublicKey
 }
 
+// Well-known errors
+var (
+	ErrAssertionNotFound       = errors.New("assertion not found")
+	ErrAssertionNotSuperseding = errors.New("assertion does not supersede current one")
```

Comment:
I suggest `s/Assertion//` here, because we're already in the context of the `asserts` package here (e.g. `asserts.ErrNotFound` seems about right).